src/setup/txt_joybinput.c

```diff
@@ -228,6 +299,43 @@ static void OpenPromptWindow(txt_joystick_input_t *joystick_input)
     SDL_JoystickEventState(SDL_ENABLE);
 }
 
+static void OpenPromptWindowGamepad(txt_joystick_input_t *joystick_input)
+{
+    txt_window_t *window;
+    SDL_GameController *gamepad;
+
+    // Silently update when the shift button is held down.
+
+    joystick_input->check_conflicts = !TXT_GetModifierState(TXT_MOD_SHIFT);
+
+    if (SDL_Init(SDL_INIT_GAMECONTROLLER) < 0)
```

Comment:
Should this be `SDL_InitSubsystem` ?

Reply:
`SDL_InitSubsytem` and `SDL_Init` are interchangeable in SDL2. Also, I was following the convention of the project which is to use `SDL_Init`.